migrations/20170508160729_app_workload_points.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('workload_points', function (table) {
+    table.increments('id')
+    table.timestamp('effective_from').defaultTo(knex.fn.now())
+    table.timestamp('effective_to')
+  }).catch(function (error) {

```

Comment:
Needs to have columns added for the adjustable config values.

001 Workload Points Modification Business Rules contains the values for this one. Lets have a chat about this one tomorrow morning as I think we may be able to do something more the design of this.